command/resource/resource.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package resource
+
+import (
+	"github.com/mitchellh/cli"
+
+	"github.com/hashicorp/consul/command/flags"
+)
+
+func New() *cmd {
+	return &cmd{}
+}
+
+type cmd struct{}
+
+func (c *cmd) Run(args []string) int {
+	return cli.RunResultHelp
+}
+
+func (c *cmd) Synopsis() string {
+	return synopsis
+}
+
+func (c *cmd) Help() string {
+	return flags.Usage(help, nil)
+}
+
+const synopsis = "Interact with Consul's resources"
+const help = `
+Usage: consul resource <subcommand> [options]
+
+This command has subcommands for interacting with Consul's resources.
+Here are some simple examples, and more detailed examples are available
+in the subcommands or the documentation.
+
+Read a resource:
+
+$ consul resource read [type] [name] -partition=<default> -namespace=<default> -peer=<local> -consistent=<false> -json
```

Comment:
Did we miss instantiating the `consistent` flag?

Reply:
It is best to stick with `stale` flag if other CLI commands are using `stale` instead of `consistent`. can you check?

Reply:
right, `stale` is a http flag already, I think we need to stick with it instead of using a new name

https://github.com/hashicorp/consul/blob/4f9955d91e1f7cdab68ab407d4789e1339ff6bfa/command/flags/http.go#L73

Reply:
Just updated the code